servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherAsBlockingIterable.java

```diff
@@ -49,7 +49,7 @@
  * @param <T> Type of items emitted by the {@link Publisher} from which this {@link BlockingIterable} is created.
  */
 final class PublisherAsBlockingIterable<T> implements BlockingIterable<T> {
```

Comment:
A more detailed comment on `itemsToRequestNext` describing problem with blocking `Iterator` might be helpful.

Reply:
I'll move this comment to https://github.com/apple/servicetalk/issues/2205.